internal/cmd/commands/clientagentcmd/addtoken.go

```diff
@@ -48,10 +48,8 @@ func addToken(ctx context.Context, apiClient *api.Client, port uint) (*api.Respo
 	client.RetryWaitMin = 100 * time.Millisecond
 	client.RetryWaitMax = 1500 * time.Millisecond
 
-	// TODO (ICU-13140): Until we release the client agent, do not retry attempts
-	// to connect to the client agent since it adds a noticeably long delay to
-	// the command.
-	client.RetryMax = 0
+	// Explicitly setting this to 1, since this runs after every command and we don't want any delays
+	client.RetryMax = 1
```

Comment:
This retry would potentially add 1.5 seconds to every CLI command when ferry isn't running on the system, no? I wonder if we should leave this at 0?

Reply:
Sure, I'll boop it back to 0